### DISCUSSION TITLE
I can't install both ocaml-lsp and ocamlformat at the same time

### DISCUSSION THREAD

**Discussion by brandoncc** [mason-org/mason.nvim#1388] [Q&A] [answered]
I can install either of these, but when I try to install the second one the installation fails with:

```
"/Users/brandoncc/.local/share/nvim/mason/bin/ocamllsp" is already linked.
```

It seems both of these try to install `ocamllsp`, and only one of them is allowed to. Is this a known problem and is there a known solution?

UPDATE: when I `ls -la` in `/Users/brandoncc/.local/share/nvim/mason/bin`, I see one of the entries is:

```
ocamllsp -> /Users/brandoncc/.local/share/nvim/mason/packages/ocamlformat/bin/ocamlformat
```

That seems like a mistake.

I suspect the issue is the `ocamllsp` [here](https://github.com/mason-org/mason-registry/blob/edfceaf8b2021ae39fb7b7ae0720a7fd99efefff/packages/ocamlformat/package.yaml#L18), but I haven't figured out how to use a fork of `mason-registry` where I can commit a change and test it.

**Comment by williamboman** [comment, marked as answer]
Hello! Oh yeah that does not look right at all. While locally testing packages in the registry is possible, it's unfortunately a bit complicated and involves some manual setup first (if you're interested I'd look into [this GitHub action](https://github.com/mason-org/actions/tree/main/tests) which is used to run package tests in CI). Just changing the bin name should be fine, I'll merge a fix immediately and try to think of improved linting rules, because this has happened before (probably due to copy pasting).

**Reply by brandoncc** [reply]
I confirmed the problem is solved. I would have opened a PR for you but I wasn't sure it was the solution, thank you for fixing it.

Side note, thank you for maintaining this amazing tool!
